distribution/docs/src/main/resources/_contents/_configuring/configuring-managed-service-factory-bundles-contents.adoc

```diff
@@ -3,15 +3,14 @@
 
 Services that are created using a Managed Service Factory can be configured using `.config` files as well.
 These configuration files, however, follow a different naming convention.
-The files must start with the Managed Service Factory PID, be followed by a unique identifier and have a `.config` extension.
-For instance, assuming that the Managed Service Factory PID is `org.codice.ddf.factory.pid` and two instances of the service need to be configured, files `org.codice.ddf.factory.pid.<UNIQUE ID 1>.config` and `org.codice.ddf.factory.pid.<UNIQUE ID 2>.config` should be created and added to `<${branding}_HOME>/etc`.
+The files must start with the Managed Service Factory PID, be followed by a dash and an unique identifier, and have a `.config` extension.
+For instance, assuming that the Managed Service Factory PID is `org.codice.ddf.factory.pid` and two instances of the service need to be configured, files `org.codice.ddf.factory.pid-<UNIQUE ID 1>.config` and `org.codice.ddf.factory.pid-<UNIQUE ID 2>.config` should be created and added to `<${branding}_HOME>/etc`.
 
 The unique identifiers used in the file names have no impact on the order in which the configuration files are processed.
 No specific processing order should be assumed.
 Also, a new service will be created and configured every time a configuration file matching the Managed Service Factory PID is added to the directory, regardless of the _unique id_ used.
 
-These configuration files must also contain a `service.factoryPid` property set to the factory PID (without the sequential number).
-They should not however contain the `service.pid` property.
+Any `service.factoryPid` and `service.pid` values in these `.config` files will be overriden by the values parsed from the file name, so `.config` files should not contain these properties.
 
 ===== File Format
 
```

Comment:
❗️ If this table describes the attributes that the admin has to configure, then we shouldn't list `service.pid` or `service.factoryPid` since those are set by Karaf. If it describes the attributes _after_ the files have been updated, we should add the file path attribute added.